hydra_python_core/doc_maker.py

```diff
@@ -1,10 +1,7 @@
 """Contsructor to take a Python dict containing an API Documentation and
 create a HydraDoc object for it
 """
-import re
-import json
 from pyld import jsonld
-import requests
```

Comment:
This import is internally required by the pyld. According to it's docs: 

> Defining a dependency on pyld will not pull in Requests or aiohttp. If you need one of these for a Document Loader then either depend on the desired external library directly or define the requirement as PyLD[requests] or PyLD[aiohttp].

So, we might want to modify either requirements.txt or leave it here.

Reply:
Ok, in that case this was a false positive! Thanks for pointing this out :D